base/doc/ltnews40.tex

```diff
@@ -442,47 +571,43 @@ \subsection{\pkg{doc}:\ \cs{PrintDescribeMacro} in preamble}
 \githubissue{1000}
 
 
-\section{Improvement to \XeTeX\ \cs{showhyphens}}
-When using \cs{showhyphens} with \XeTeX, missing character warnings
-would be generated for any character not in Latin Modern. This has been
-corrected and the warnings are suppressed.
-%
-\githubissue{1380}
-
-
-\section{Avoid code duplication in rollback}
+\subsection{Avoid code duplication in rollback}
 
-When the kernel uses \cs{AddToHook} in a region that might be 
-rolled back (which happens in a few places) and a document requests a
-rollback, then we have the situation that the hook already contains code to which we added the same
-(or slightly different) code during the
-rollback; this results in code duplication or, worse, in errors. This has
-now been corrected by dropping any such code chunk (if there is one) prior to
-adding the rollback code.
+When the kernel uses \cs{AddToHook} in a region that might be rolled
+back (which happens in a few places) and a document requests a
+rollback, then we have the situation that the hook already contains
+code to which we added the same (or slightly different) code during
+the rollback; this results in code duplication or, worse, in
+errors. This has now been corrected by dropping any such code chunk
+(if there is one) prior to adding the rollback code.
 %
 \githubissue{1407}
 
-\section{Passing template keys using \cs{KeyValue}}
 
-With the move of the template code to the kernel, some internal efficiencies
-were also made. However, there was an oversight in how passing key values from
-one setting to another was implemented, meaning that using \cs{KeyValue} could
-result in an infinite loop. This has now been fixed.
+\subsection{Passing template keys using \cs{KeyValue}}
+
+With the move of the template code to the kernel, some internal
+efficiencies were also made. However, there was an oversight in how
```

Comment:
Not sure how this would be correct, but "efficiencies were also made" sounds wrong to me.

Reply:
Online Cambridge Dictionary has an entry for "efficiency" as countable noun

> something that is done to avoid wasting time, energy, or money: [...]
> https://dictionary.cambridge.org/us/dictionary/english/efficiency

Reply:
wasn't my text -- @josephwright ?  (even if correct according to @muzimuzhi  I think most non-native speakers stumble)

Reply:
It is correct, but if it's an issue we can change - perhaps
```suggestion
\subsection{Passing template keys using \cs{KeyValue}}

With the move of the template code to the kernel, internal structures
were reviewed to improve efficiency. However, there was an oversight in how
```